.flake8

```diff
@@ -5,3 +5,4 @@ ignore =
 
 max-line-length=88
 max-complexity=39
+min_python_version = 3.6.0
```

Comment:
The `python_requires` option in `setup.cfg` will be used so this isn't necessary.
https://github.com/asottile/flake8-typing-imports#configuration

Reply:
👍 Right, changing from ``~=`` to ``>=`` is necessary enough. (and this alone does not work because the value in setup.cfg override it.)